lib/Secrets/extensionplugins.h

```diff
@@ -51,6 +51,12 @@ class SAILFISH_SECRETS_API EncryptionPlugin
     virtual Sailfish::Secrets::EncryptionPlugin::EncryptionType encryptionType() const = 0;
     virtual Sailfish::Secrets::EncryptionPlugin::EncryptionAlgorithm encryptionAlgorithm() const = 0;
 
+    virtual bool supportsLocking() const;
```

Comment:
The more I look at this, the more I think there should be a common base class for these plugin classes. Does not need to be part of this PR, but would definitely improve the code maintainability.

Reply:
Right, let's do that in a separate PR